tpm_test.go

```diff
@@ -476,7 +476,7 @@ func TestConnectToDefaultTPM(t *testing.T) {
 
 func TestSecureConnectToDefaultTPM(t *testing.T) {
 	SetOpenDefaultTctiFn(func() (io.ReadWriteCloser, error) {
-		return tpm2.OpenMssim(*mssimHost, *mssimTpmPort, *mssimPlatformPort)
+		return tpm2.OpenMssim("", *mssimPort, *mssimPort + 1)
```

Comment:
Is the port, port+1 for the command and platform server ports the default for tpm2-simulator? (Because I see that only mssimPort is used when starting the simulator)

Reply:
The TPM simulator opens the TPM interface on the port provided on the commandline (tcp port 2321 by default), and the platform port is always started on the next port number (tcp port 2322 by default). The fact that go-tpm2 requires both ports to be provided is probably a bug - it should just take a single port number and assume that the platform port will use the next port number.